src/full/Agda/Interaction/Options.hs

```diff
@@ -27,6 +27,7 @@ module Agda.Interaction.Options
     , mapFlag
     , usage
     , defaultLibDir
+    , allowExec
```

Comment:
Do you need this to be exported? I couldn't find it used anywhere.

Reply:
Don't think I do. Removed the export.